### ISSUE TITLE
Crashed when a live viewer closed their browser

### ISSUE THREAD

**Issue by ebenezerdon** [codesandbox/codesandbox-client#4250]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>
nothing on my end. I was live-sharing

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [https://codesandbox.io/s/animations-3yvgj?file=/src/Example.tsx]() (link)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 83.0.4103 | Windows 10           | 3f53157da          |

  **Route:**
  https://codesandbox.io/s/animations-3yvgj?file=/src/Example.tsx

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'username' of undefined
  TypeError: Cannot read property 'username' of undefined
    at https://codesandbox.io/static/js/app.719d61325.js:571:997262
    at Array.map (<anonymous>)
    at Cu (https://codesandbox.io/static/js/app.719d61325.js:571:996784)
    at ua (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1735529)
    at Ka (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1743794)
    at xs (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1781982)
    at ml (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1766368)
    at hl (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1766293)
    at il (https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1763621)
    at https://codesandbox.io/static/js/common.8f6fa98d0.chunk.js:1:1721804
  
    in Cu
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in c
    in t
  ```

  </details>

**Comment by CompuIves** [comment]
This one should be fixed by #4176

**Comment by lbogdan** [comment]
Closing this, it should already be fixed by #4176 .
